data/human/deep jobs.txt

```diff
@@ -406,24 +412,20 @@ mission "Secure Transport Job (Secret) [0]"
 		"reputation: Deep Security" >= -50
 	source
 		attributes "deep"
-		not attributes "station" "moon"
```

Comment:
Why have these been removed from the Source? I understand the validity of also excluding them from the destination, but I would expect they need to remain in the source as well.

Reply:
Is there a reason why they can't be offered from moons or stations?

Reply:
Mostly just none of the texts make sense for them.